samples/scripts/settings.ps1

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/pwsh
+
+# In case you are running into issues with Windows execution policies, you can unblock the *.ps1 files via:
+# Unblock-File -Path ./*.ps1
+
+Write-Host "Using parameters specified in settings.ps1."
+
+# always edit this parameter before running the scripts
+$name = "testnetheritescripts"
+
+# review these parameters before running the scripts
+$location = "westeurope"
+$storageSku = "Standard_LRS"
+
+# optionally, customize the following parameters
+# to use different names for resource group, namespace, function app, storage account, and plan
+$groupName = $name
+$nameSpaceName = $name
+$functionAppName = $name
+$storageName = $name
+$planName = $name
+
+if (($name -eq "globally-unique-lowercase-alphanumeric-name-with-no-dashes")) {
```

Comment:
I think this test condition should be updated to `($name -eq "testnetheritescripts")` ... unless I am missing something.

Reply:
The other way round - did not changed that back to your setup after my last test